packages/block-library/src/buttons/test/edit.native.js

```diff
@@ -271,4 +274,124 @@ describe( 'Buttons block', () => {
 			} )
 		);
 	} );
+
+	describe( 'color customization', () => {
+		it( 'sets a text color', async () => {
+			// Arrange
+			const screen = await initializeEditor();
+			await addBlock( screen, 'Buttons' );
+
+			// Act
+			const buttonsBlock = getBlock( screen, 'Buttons' );
+			fireEvent.press( buttonsBlock );
+
+			// Trigger onLayout for the list
```

Comment:
Given we have to do this so often, I question the value of placing this inline comment. Should we instead consider renaming the helper to more clearly communicate intent/rationale/outcome. WDYT?